common/client-core/config-types/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "nym-client-core-config-types"
```

Comment:
Just a personal taste comment: imho it's nice to group related crates in a subdir, but nested crates feels iffy. Just like nested workspaces

Reply:
Yeah, I'm not the biggest fan of it but couldn't think of anything much better without other drawbacks